source/transactions/tests/README.rst

```diff
@@ -317,6 +329,13 @@ A ``readConcern.afterClusterTime`` value of ``42`` in a command-started event
 is a fake cluster time. Drivers MUST assert that the actual command includes an
 afterClusterTime.
 
+recoveryToken
+^^^^^^^^^^^^^
+
+A ``recoveryToken`` value of ``42`` in a command-started event is a fake
+recovery token. Drivers MUST assert that the actual command includes a
+"recoveryToken" field.
```

Comment:
Consider:

> Drivers MUST assert that the actual command includes a
"recoveryToken" field equal to the recovery token stored in the ClientSession

It seems a little loose otherwise.

Reply:
I agree that extra validation like this may be useful, but it doesn't seem appropriate for the `42` placeholder.

Perhaps we can borrow language from the [logical session ID](https://github.com/mongodb/specifications/blob/master/source/transactions/tests/README.rst#logical-session-id) assertion, which uses a reference to a session object and discusses how it should be interpreted for the assertion.